src/ui/impl/webview/webview.hpp

```diff
@@ -13,6 +13,7 @@ namespace Soundux
             std::shared_ptr<Tray::Tray> tray;
             std::shared_ptr<Webview::Window> webview;
             void changeSettings(const Settings &newSettings) override;
+            bool hideOnStartup = false;
```

Comment:
This should not be here, it should be in the settings.
I'll add support for this later myself